### PR TITLE
perf(Dashboard): Make scroll bars always visible

### LINKED ISSUE #2079: [Feature Request]: Make horizontal scroll bar always visible
### Description

Right now the horizontal scroll  bar is only at bottom.

If we can get an option like the pagination to put additional scroll bar at top it would be great. 

### Describe the solution you'd like

_No response_

### Describe alternatives you've considered

_No response_

### Complementary informations

- [ ] Is this feature already implemented in the default WebUI?

### PR DESCRIPTION
**Pull request by Larsluph**
Fixes #2079

### CODE REVIEW

**Review by WDaan — APPROVED** [review]

File: src/components/Dashboard/Views/Table/TableView.vue [hunk 1]
```diff
@@ -46,19 +54,30 @@ const getTorrentRowColorClass = (torrent: TorrentType) => [isTorrentSelected(tor
...
       </tr>
-    </thead>
-    <tbody>
+    </template>
+
+    <template #item="{ item: torrent }">
       <tr
-        v-for="torrent in paginatedTorrents"
-        :class="['cursor-pointer', 'selected', 'ripple-fix', getTorrentRowColorClass(torrent)]"
+        :class="['cursor-pointer', 'selected', getTorrentRowColorClass(torrent)]"
```
Comment (asrevni) [inline src/components/Dashboard/Views/Table/TableView.vue:80]: This reverts to ripple effect issue found with Safari browser. Any reason for ommiting `ripple-fix`?

Comment (Larsluph) [reply src/components/Dashboard/Views/Table/TableView.vue:80]: I checked the code and bo class with that name exists in our CSS. I don't remember having fixed that issue...

Comment (asrevni) [reply src/components/Dashboard/Views/Table/TableView.vue:80]: #1864. It's still a valid workaround.

Comment (Larsluph) [reply src/components/Dashboard/Views/Table/TableView.vue:80]: Well I may have messed something up while looking for it, thanks for the review!

Comment (asrevni) [reply src/components/Dashboard/Views/Table/TableView.vue:80]: Thank you too!

**Pull request merged**